src/common/hooks/useUnreceive.js

```diff
@@ -0,0 +1,51 @@
+import { some } from 'lodash';
+import { useMutation } from 'react-query';
+
+import { useOkapiKy } from '@folio/stripes/core';
+import { ITEM_STATUS } from '@folio/stripes-acq-components';
+
+import { RECEIVE_API } from '../constants';
+
+export const useUnreceive = (options = {}) => {
+  const ky = useOkapiKy();
+
+  const { mutateAsync } = useMutation({
+    mutationFn: (pieces) => {
+      const selectedPieces = pieces
```

Comment:
you don't need to duplicate code, there is already a tool to unreceive https://github.com/folio-org/ui-receiving/blob/master/src/common/utils/receive.js#L5. Just use it (with adapter to use `ky` instead of mutator).